### ISSUE TITLE
META-002: add accessModeSufficient=visual

### ISSUE THREAD

**Issue by madeleinerothberg** [w3c/epub-specs#978]
In technique "META-002: Identify sufficient access modes" the book can be read visually, textually, or both together. Add accessModeSufficient for visual alone.

**Comment by madeleinerothberg** [comment]
After further discussion, we have realized that this resource should not be listed as accessible through Visual alone. We will clarify that the resource is accessible through text + visual because it has images plus text; the fact that one way you can read text is with your eyes does not mean that true text should be categorized as "visual" in the same way the fact that you can hear the text read aloud (by TTS)  does not make it "auditory." It is text.

**Comment by mattgarrish** [comment]
Proposed Solution

Add a note explaining why visual is not listed as a sufficient access mode.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section5) on 2020-12-10)

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 5. META-002: add `accessModeSufficient=visual` 

_See github issue [#978](https://github.com/w3c/epub-specs/issues/978)._




**Avneesh Singh:** There was a suggestion over on github that when there is text in a book, this should satisfy `accessmodesufficient=visual`.  

> **Proposed resolution: We will not add `accessmodesufficient=visual` for text content, instead we will clarify why it should not be done.** *(Avneesh Singh)*

**Avneesh Singh:** but discussion revealed that text can be consumed via multiple modes.  
… so instead, Matt G proposed that there should be a note explaining this concept, and why accessmodesufficient=visual does not apply.  
… and why such cases should be covered by `accessmodesufficient=textual`.  

**Charles LaPierre:** if a book has both images and text and the images are described via alt-text, then `accessmodesufficient=textual` works, but also a`ccessmodesufficient=textual, visual`.  

**Avneesh Singh:** to clarify, `accessmodesufficient` means that the content of the book can be consumed via ALL the following modes.  
… so the comma in that attribute means logical AND.  

**Bill Kasdorf:** somewhere we need to be clear on: 1. that two values is AND (not OR), and 2. that `accessmodesufficient` property is repeatable.  

**Avneesh Singh:** a lot of this also depends on how you are presenting it.  
… e.g. if you are presenting this information in JSON vs XML.  
… in JSON this would all be in a list.  

> **Proposed resolution: We will not add `accessmodesufficient=visuala for text content, instead we will clarify why it should not be done.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Charles LaPierre:* +1.

> *Laura Brady:* +1.

> *Bill Kasdorf:* +1.

> *Juliette McShane:* +1.

> *Ben Schroeter:* 0.

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section5) on 2020-12-10)

List of resolutions:

- ***[Resolution No. 3](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#resolution3):  We will not add `accessmodesufficient=visual` for text content, instead we will clarify why it should not be done.***

<details><summary><i>View the transcript</i></summary>

### 5. META-002: add `accessModeSufficient=visual` 

_See github issue [#978](https://github.com/w3c/epub-specs/issues/978)._




**Avneesh Singh:** There was a suggestion over on github that when there is text in a book, this should satisfy `accessmodesufficient=visual`.  

> **Proposed resolution: We will not add `accessmodesufficient=visual` for text content, instead we will clarify why it should not be done.** *(Avneesh Singh)*

**Avneesh Singh:** but discussion revealed that text can be consumed via multiple modes.  
… so instead, Matt G proposed that there should be a note explaining this concept, and why accessmodesufficient=visual does not apply.  
… and why such cases should be covered by `accessmodesufficient=textual`.  

**Charles LaPierre:** if a book has both images and text and the images are described via alt-text, then `accessmodesufficient=textual` works, but also a`ccessmodesufficient=textual, visual`.  

**Avneesh Singh:** to clarify, `accessmodesufficient` means that the content of the book can be consumed via ALL the following modes.  
… so the comma in that attribute means logical AND.  

**Bill Kasdorf:** somewhere we need to be clear on: 1. that two values is AND (not OR), and 2. that `accessmodesufficient` property is repeatable.  

**Avneesh Singh:** a lot of this also depends on how you are presenting it.  
… e.g. if you are presenting this information in JSON vs XML.  
… in JSON this would all be in a list.  

> **Proposed resolution: We will not add `accessmodesufficient=visual` for text content, instead we will clarify why it should not be done.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Charles LaPierre:* +1.

> *Laura Brady:* +1.

> *Bill Kasdorf:* +1.

> *Juliette McShane:* +1.

> *Ben Schroeter:* 0.

> ***Resolution #3: We will not add `accessmodesufficient=visual` for text content, instead we will clarify why it should not be done.***

</details>
